### PR TITLE
Feature/automate quotes

### PR DESCRIPTION
**Pull request by MattIzSpooky**
Closes https://github.com/MattIzSpooky/tf2.rest/issues/5
Closes https://github.com/MattIzSpooky/tf2.rest/issues/2

### CODE REVIEW

File: responses/response.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	SPY      = "spy"
+)
+
+var ALL []Response
+
+func Setup() {
+	// Attack classes
+	//ALL = append(ALL, scoutResponses...)
+	ALL = append(ALL, soldierResponses...)
+	//ALL = append(ALL, pyroResponses...)
```
Comment (MattIzSpooky) [inline responses/response.go:31]: should be commented back in

File: responses/response.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	MEDIC    = "medic"
+	SNIPER   = "sniper"
+	SPY      = "spy"
+)
+
+var ALL []Response
+
+func Setup() {
+	// Attack classes
+	//ALL = append(ALL, scoutResponses...)
```
Comment (MattIzSpooky) [inline responses/response.go:29]: comment back in

File: codegen/scraper.go [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+		var condition string
+
+		table.Children().Find("li").Each(func(i int, listElement *goquery.Selection) {
+			conditionElement := listElement.Find("ul")
+
+			if conditionElement.Length() == 1 {
+				tempCondition := strings.TrimSpace(listElement.Contents().Not("ul").Text())
+
+				if tempCondition == "Rare" {
+					context = context + " (" + tempCondition + ")"
```
Comment (MattIzSpooky) [inline codegen/scraper.go:106]: This doesn't work. Has to be fixed

**Review by Captain-Hippie — APPROVED** [review]

**Comment by MattIzSpooky** [comment]
TODO:
- [ ] Make error handling better.
- [ ] scraper.go line 106 does not work.
- [ ] Comment in stuff

File: codegen/template.go [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+// {{ .Timestamp }}
+// using data from
+// {{ .URL }}
+package responses
+
+var {{ .Class }}Responses = []Response{
+{{- with .Responses }}
+	{{ range . }}
+	{
+		Class: 		{{ .Class | ToUpper }},
```
Comment (MattIzSpooky) [inline codegen/template.go:28]: spacing

**Pull request merged**